collector/perf_linux.go

```diff
@@ -306,8 +327,13 @@ func NewPerfCollector(logger log.Logger) (Collector, error) {
 		collector.perfSwProfilers[cpu] = &swProf
 		collector.swProfilerCPUMap[&swProf] = cpu
 
-		cacheProf, err := perf.NewCacheProfiler(-1, cpu)
-		if err != nil {
+		cacheProfilers := perf.L1DataReadHitProfiler& perf.L1DataReadMissProfiler& perf.L1DataWriteHitProfiler& perf.L1InstrReadMissProfiler& perf.InstrTLBReadHitProfiler& perf.InstrTLBReadMissProfiler& perf.LLReadHitProfiler& perf.LLReadMissProfiler& perf.LLWriteHitProfiler& perf.LLWriteMissProfiler& perf.BPUReadHitProfiler& perf.BPUReadMissProfiler
```

Comment:
I may consider using [`AllCacheProfilers`](https://github.com/hodgesds/perf-utils/blob/master/cache_profiler.go#L18) for clearer code but that may enable some profilers that aren't used and cause more multiplexing of perf events on the CPU.